### ISSUE TITLE
Loop back path completed items colors

### ISSUE THREAD

**Issue by madhurrya** [sartography/spiff-arena#1944]
In this diagram it looks bit odd that some completed items are shown in gray and some are in white, when it loops back. 
Model : https://sample.spiffcrm.com/process-models/misc:qa:madhu-testing:lane-test-1

![Image](https://github.com/user-attachments/assets/963046a2-992f-4e0f-8d43-f84f57ec983f)

**Comment by jasquat** [comment]
Yeah this is awkward and always has been like this. This is because those white tasks are now in the `FUTURE` since that is the most recent instances of them. We could color future tasks some other color as a quick "fix" and I believe the idea of having an instance counter has been brought up before to help show this.
